File: JULIA/Cap04/c04ex01.jl
# c04ex01.jl

print("Entre o 1o. valor numerico: ")
a = parse(Int, readline())

print("Entre o 2o. valor numerico: ")
b = parse(Int, readline())

x = a + b

if (x > 10)
  println("Resultado = $x")
end

println()
print("Tecle <Enter> para encerrar... ")
readline()

File: CSHARP/Cap04/c04ex01.cs
// c04ex01.cs

using System;

class Program
{
    static void Main(string[] args)
    {
        int A, B, X;

        Console.Write("Entre o 1o. valor numerico: ");
        A = int.Parse(Console.ReadLine());

        Console.Write("Entre o 2o. valor numerico: ");
        B = int.Parse(Console.ReadLine());

        X = A + B;

        if (X > 10)
            Console.WriteLine("Resultado = " + X);

        Console.Write("\nAperte <Enter> para encerrar... ");
        Console.ReadLine();
    }
}
